src/tests/test_utils/common_test_utils/include/common_test_utils/data_utils.hpp

```diff
@@ -465,6 +465,27 @@ void inline fill_data_random<InferenceEngine::Precision::BF16>(InferenceEngine::
 }
 OPENVINO_SUPPRESS_DEPRECATED_END
 
+inline void fill_random_string(std::string* dst,
+                               const size_t size,
+                               const size_t len_range = 10lu,
+                               const size_t start_from = 0lu,
+                               const int seed = 1) {
```

Comment:
non template functions should have their implementations in cpp files

Reply:
Done